VoodooI2CHID/VoodooI2CMultitouchHIDEventDriver.cpp

```diff
@@ -614,17 +614,24 @@ IOReturn VoodooI2CMultitouchHIDEventDriver::parseElements() {
 
         if (element->getUsage() == 0)
             continue;
-
-        if (element->conformsTo(kHIDPage_Digitizer, kHIDUsage_Dig_Pen)
-            || element->conformsTo(kHIDPage_Digitizer, kHIDUsage_Dig_TouchScreen)
-            || element->conformsTo(kHIDPage_Digitizer, kHIDUsage_Dig_TouchPad)
-            || element->conformsTo(kHIDPage_Digitizer, kHIDUsage_Dig_DeviceConfiguration)
-            )
+        
+        if (element->conformsTo(kHIDPage_Digitizer, kHIDUsage_Dig_DeviceConfiguration)
```

Comment:
The condition here is a bit hairy and hard to read. Can we simplify it a bit?

Reply:
If it's not possible, can we add a comment documenting what the if-statement does and break it down logically?

Reply:
Yeah, I wasn't too sure about that. I'll probably break it a part a bit, and can definitely add comments.